ob1k-core/src/main/java/com/outbrain/ob1k/server/netty/NettyRequest.java

```diff
@@ -198,24 +198,28 @@ public String getContextPath() {
 
   @Override
   public String getCookie(final String cookieName) {
+    final Cookie cookie = getNettyCookie(cookieName);
+    return cookie != null ? cookie.value() : null;
+  }
+
+  public Cookie getNettyCookie(final String cookieName) {
```

Comment:
just remember that once people are gonna use directly the Cookie object, migration path to newer versions of Netty will be harder

Reply:
You are right. I will close this pull request and parse the cookie directly from the header.